frontend/src/TooljetDatabase/Table/index.jsx

```diff
@@ -242,6 +269,14 @@ const Table = ({ openCreateRowDrawer }) => {
           openCreateRowDrawer={openCreateRowDrawer}
           totalRecords={totalRecords}
           fetchTableData={fetchTableData}
+          filters={queryFilters}
+          sortFilters={sortFilters}
+          dataLoading={loading}
+          selectedTable={selectedTable}
+          handleBuildFilterQuery={handleBuildFilterQuery}
+          buildPaginationQuery={buildPaginationQuery}
+          resetFilterQuery={resetFilterQuery}
+          tableDataLength={tableData.length}
```

Comment:
Can we please directly derive these values from the `useContext` inside `TableFooter`